socorro/tests/external/es/test_supersearch.py

```diff
@@ -463,17 +463,11 @@ def test_get_with_combined_operators(self, es_helper):
         now = utc_now()
         crashstorage = self.build_crashstorage()
         api = SuperSearchWithFields(crashstorage=crashstorage)
-        sigs = (
```

Comment:
This test started failing after I fixed the code. I don't like tests that define things far away from where you need to look to understand what the test is doing, so I moved these to be inline even though it becomes repeated strings.